CommonTools/UtilAlgos/interface/EventSelectorAdapter.h

```diff
@@ -17,6 +17,14 @@
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "HLTrigger/HLTcore/interface/defaultModuleLabel.h"
```

Comment:
if needed in this package, this file will need to be moved to some place in FWCore